components/uuf-core/src/test/java/org/wso2/carbon/uuf/core/APITest.java

```diff
@@ -52,7 +54,7 @@ public void testSendError() {
         Assert.assertThrows(IllegalArgumentException.class, () -> API.sendError(500, ""));
 
         HttpErrorException exception = Assert.expectThrows(HttpErrorException.class,
-                                                           () -> API.sendError(500, "Some internal server error!"));
```

Comment:
This formatting cannot be reverted as it violates the rule of 120 characters in a line